<?xml version="1.0" encoding="UTF-8"?>
<!--
DFF - An Open Source Digital Forensics Framework
Copyright (C) 2011 ArxSys

This program is free software, distributed under the terms of
the GNU General Public License Version 2. See the LICENSE file
at the top of the source tree.

See http://www.digital-forensic.org for more information about this
project. Please do not directly contact any of the maintainers of
DFF for assistance; the project provides a web site, mailing lists
and IRC channels for your use.

Author(s):
Christophe Malinge cma@digital-forensic.org
-->
<ui version="4.0">
 <class>FindFile</class>
 <widget class="QGroupBox" name="FindFile">
  <property name="windowTitle">
   <string/>
  </property>
  <property name="title">
   <string>Find file</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pattern:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Syntax:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="filterPatternLineEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="filterSyntaxComboBox">
           <item>
            <property name="text">
             <string>Regular expression</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fixed string</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Wildcard</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Attribute:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="filterColumnComboBox">
           <item>
            <property name="text">
             <string>Name</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Size</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Date</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="sortCaseSensitivityCheckBox">
       <property name="text">
        <string>Case sensitive sorting</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="filterCaseSensitivityCheckBox">
       <property name="text">
        <string>Case sensitive filter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
